RobotsWantedLeague/Models/Robot.cs

```diff
@@ -1,7 +1,14 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
 namespace RobotsWantedLeague.Models;
 
 public class Robot{
-    public int Id { get; set;}
```

Comment:
This is important.  I add custom annotation to make sure the robot ID matches the _id in mongodb.